lib/mix/tasks/phx.gen.auth/hasing_library.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Mix.Tasks.Phx.Gen.Auth.HashingLibrary do
```

Comment:
I think there is type on the file name 😅 

`%s/hasing/hashing`

Reply:
Thanks for catching that @feliperenan! Fixed in https://github.com/phoenixframework/phoenix/pull/4077/commits/ff10072354005220223402eef5b07e0f45704e72.